.github/workflows/deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    env:
+      IMAGE_NAME: eu.gcr.io/${{ secrets.GCP_PROJECT_ID }}/turnilo
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Authenticate on GCP
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - name: Configure Docker
+        run: gcloud auth configure-docker --quiet
+
+      - name: Build Docker image
+        run: docker build . -t $IMAGE_NAME
```

Comment:
I'd recommend adding tag to image name, otherwise docker just slaps in the default `latest` tag, and subsequent deploys will overwrite each other. If thats the desired behavior, I'd suggest setting `latest` tag explicitly for self-documenting

Reply:
It's a bit more difficult - I thought that the image with implicit "latest" tag overwrites image in the container registry (costs optimisation - we don't need previous images) but I was wrong, all images are kept - the last one gets "latest" tag.

What do you think about GITHUB_SHA instead of "latest" (and https://github.com/sethvargo/gcr-cleaner for costs optimisation)?